### PR TITLE
Fix broken weblink and typos

### PR DESCRIPTION
**Pull request by git-taufiq**

### CODE REVIEW

**Comment by sebwolf-de** [comment]
Hi,
good to see these fixes :-)
Just a small remark: We sometimes cite with _author_(_year_) and sometimes with (_autor_, _year). Maybe we can also streamline this.

**Comment by daisy20170101** [comment]
it should be https://github.com/SeisSol/SeisSol   :-P

**Review by git-taufiq — COMMENTED** [review]
updated Github link

File: Documentation/introduction.rst [hunk 1]
```diff
@@ -22,13 +22,12 @@ some of the largest supercomputers worldwide in a pioneering simulation of the 1
...
+stepping (Uphoff et al.,2017), resolving the 2004
 Sumatra-Andaman earthquake including complex splay fault geometries. The
 paper won the prestigious “Best Paper Award” of the International
 Supercomputing Conference (SC17). SeisSol results imply that
-acknowledging geometrical complexity, realistic fault properties and
+acknowledging geometrical complexity, realistic fault properties, and
 velocity models affect not only earthquake source dynamics but the
 synthetic ground shaking crucially. The software package is available to
-the community as an open-source distribution
-(`www.github.com/SeisSol/SeisSol <www.github.com/SeisSol/SeisSol>`__).
```
Comment (Thomas-Ulrich) [inline Documentation/introduction.rst:33]: This is a clickable link. See here:
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages
Please revert this change.

Comment (git-taufiq) [reply Documentation/introduction.rst:33]: the previous version lead to this broken link if we clicked it: https://github.com/SeisSol/SeisSol/blob/master/Documentation/www.github.com/SeisSol/SeisSol

Comment (Thomas-Ulrich) [reply Documentation/introduction.rst:33]: Here is an examples which works:
```
... using Gmsh are provided
at this `link <https://github.com/SeisSol/SeisSol/tree/master/preprocessing/meshing/gmsh_example>`_.
```

Comment (sebwolf-de) [reply Documentation/introduction.rst:33]: > If the link text should be the web address, you don’t need special markup at all, the parser finds links and mail addresses in ordinary text.

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links

Comment (Thomas-Ulrich) [reply Documentation/introduction.rst:33]: Ok then, the change is fine!

**Review by Thomas-Ulrich — APPROVED** [review]

**Comment by daisy20170101** [comment]
Looks good

**Review by daisy20170101 — APPROVED** [review]
